command/e2e/register_validator.go

```diff
@@ -50,9 +54,28 @@ func setFlags(cmd *cobra.Command) {
 		"",
 		"the directory path where registrator validator key is stored",
 	)
+
+	cmd.Flags().StringVar(
+		&params.balance,
+		balanceFlag,
+		defaultBalance,
+		"balance which is going to be funded to the new validator account",
+	)
+
+	cmd.Flags().StringVar(
+		&params.stake,
+		stakeFlag,
+		defaultStake,
+		"stake represents amount which is going to be staked by the new validator account",
+	)
+
+	helper.RegisterGRPCAddressFlag(cmd)
```

Comment:
I think you don't need this flag.

Reply:
You are right, it was there before, so I've just moved it initially.

It is removed now: https://github.com/0xPolygon/polygon-edge/pull/1084/commits/5e73f530b365676a22ae10550a0ff9686a98c4df.